functions/private/Invoke-WinUtilMicroWin-Helper.ps1

```diff
@@ -98,6 +109,8 @@ function Remove-Features() {
             $_.FeatureName -NotLike "*Media*" -AND
             $_.FeatureName -NotLike "*NFS*" -AND
             $_.FeatureName -NotLike "*SearchEngine*" -AND
+            $_.FeatureName -NotLike "*RemoteDesktop*" -AND
+            $_.FeatureName -NotLike "*Recall*" -AND
```

Comment:
Having to add recall back in because of file explorer reversions is a bit strange.